controller/iteration_blackbox_test.go

```diff
@@ -119,17 +147,35 @@ func (rest *TestIterationREST) TestCreateChildIteration() {
 		require.Equal(t, *ci.Data.ID, *created.Data.ID)
 	})
 
-	rest.T().Run("forbidden - only space owener can create child iteration", func(t *testing.T) {
+	rest.T().Run("forbidden - user must be space-owner or collaborator to create child iteration", func(t *testing.T) {
```

Comment:
Please use nested subtests. Instead of `forbidden - reason 1` and `forbidden - reason 2` nest and group them as `forbidden/reason1` and `forbidden/reason2`

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1783/commits/80ae86445281239f604d0a038eba40ff8b347607